2018-02-01T09:59:40.949Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))
    at /home/zqh/ddos-defense-based-blockchain/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-01T09:59:40.950Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/zqh/ddos-defense-based-blockchain/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-02-01T09:59:40.950Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity, with the name 'admin' and the identifier '7d195e0cbc77f47d78f39e21ec879210cdff2f8506d5f326af245563ebfcbb69', must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/zqh/ddos-defense-based-blockchain/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-02-01T10:09:24.685Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'CO_123' as the object already exists)
    at /home/zqh/ddos-defense-based-blockchain/angular-app/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-02-01T10:09:24.686Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'CO_123' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'CO_123' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/zqh/ddos-defense-based-blockchain/angular-app/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
